src/main/java/org/bookstore/exceptions/GlobalExceptionHandler.java

```diff
@@ -35,6 +35,16 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
+    @ExceptionHandler(value = {EntityNotFoundException.class})
+    public ResponseEntity<Object> handleEntityNotFound(EntityNotFoundException ex) {
+        return new ResponseEntity<>(ex.getMessage(), HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(value = {DataProcessingException.class})
```

Comment:
you might simplify that:
```suggestion
    @ExceptionHandler(EntityNotFoundException.class)
    public ResponseEntity<Object> handleEntityNotFound(EntityNotFoundException ex) {
        return new ResponseEntity<>(ex.getMessage(), HttpStatus.NOT_FOUND);
    }

    @ExceptionHandler(DataProcessingException.class)
```